README.md

```diff
@@ -16,6 +16,44 @@ npm install @eslint/eslintrc --save-dev
 yarn add @eslint/eslintrc -D
 ```
 
+## Usage
+
+The primary class in this package is `DotCompat`, which is a utility to translate ESLintRC-style configs into dot configs. Here's how you use it inside of your `eslint.config.js` file:
```

Comment:
I was trying to riff off of "eslint dot config dot js" -> "eslint dot config" -> "dot config". Previously was "simple config," but didn't seem like a great option either in case people had trouble understanding it.

I suppose I could also just name the class `Compat` and not worry about the name, but that seems a bit risky as it would be nice to distinguish it from the eslintrc-focused functionality.

I don't like "config array" because it can also be an object or function. Open to other names, but would like to decide soon. I don't want to tie up implementation because we can't agree on a name. :)

Reply:
Random names time:

1. Volt config = efficient!
2. Bolt config = fast!
3. Morpheus config = take the blue pill!

???

Reply:
Flat config?

Reply:
"Flat config" is a good one! Describes both the lack of cascading that differentiates it from `.eslintrc.*` and associates the mental model of flattening the array members.

Reply:
Okay, let’s go with flat config.